### DISCUSSION TITLE
unfs3  v.0.10.0  compile for aarch64 - attempt

### DISCUSSION THREAD

**Discussion by tuhajbej6** [Entware/Entware#1079] [General]
Ever since the up2date unfs3 version in entware for aarch64 (RK3399 SBC in my case) is still 0.9.23 version.

https://github.com/unfs3/unfs3/releases

So I am using entware to compile this. I had to add all -dev packages in order to obtain libtirpc headers and correct the include dir for rpc.h before configure:

```
export TIRPC_CFLAGS='-I/opt/include/tirpc'
./configure
```
Now 'make' is going through that one, but when it's starting with the flex tool, I ve got this error. I have no idea what flex or lex tool are used with compile, but they are installed as required.

```
make -C Config
make[1]: Entering directory '/storage/unfs3-0.10.0/Config'
flex ./exports.l
flex: fatal internal error, exec of /media/ware4/Entware.2024.07/staging_dir/host/bin/m4 failed
make[1]: *** [Makefile:25: lex.yy.c] Broken pipe
make[1]: *** Deleting file 'lex.yy.c'
make[1]: Leaving directory '/storage/unfs3-0.10.0/Config'
make: *** [Makefile:40: Config/lib.a] Error 2
```

Am I even using the right tool ?
`flex - 2.6.4-4a - flex is a tool for generating scanners: programs which recognize lexical patterns in text`

Could anybody suggest on how to proceed with compilation ?

**Comment by ryzhovau** [comment]
Package has been updated, closing discussion for now.
